src/Sequencer4Widget.h

```diff
@@ -44,6 +44,9 @@ struct Sequencer4Widget : ModuleWidget {
         return label;
     }
 
+    Sequencer4Module* _module = nullptr;
+    void step() override;
+
```

Comment:
again, I put some Seq++ functionality into 4X4 so the undo would work here, too.